src/pages/ReimbursementAccount/ReimbursementAccountForm.js

```diff
@@ -58,9 +58,9 @@ class ReimbursementAccountForm extends React.Component {
                     onFixTheErrorsLinkPressed={() => {
                         this.form.scrollTo({y: 0, animated: true});
                     }}
-                    message={this.props.reimbursementAccount.error}
+                    message={this.props.reimbursementAccount.error || _.values(this.props.reimbursementAccount.errors)[0]}
                     isMessageHtml={this.props.reimbursementAccount.isErrorHtml}
-                    isLoading={this.props.reimbursementAccount.loading}
+                    isLoading={this.props.reimbursementAccount.loading || this.props.reimbursementAccount.isLoading}
```

Comment:
Let's just go through and refactor to sure that anything that touches the reimbursementAccount only accesses `loading` or `isLoading`. This is kinda confusing imo

Reply:
I thought about that, but I think we should keep that refactor separate since it touches many pages in the VBA flow and we are in the midst of refactoring all of them to use `Form.js` anyways, so this refactor will be undone soon.